Resolver/FormAnnotationResolver.php

```diff
@@ -60,10 +68,12 @@ class FormAnnotationResolver extends AbstractAnnotationResolver implements Annot
     public function __construct(
         FormRegistryInterface $formRegistry,
         FormFactoryInterface $formFactory,
+        RouterInterface $router,
```

Comment:
This would break the BC, so you cannot change the construct definition until next major version. This can be solved by adding a simple setter instead of modifying the method signature.